goalgorilla/open_social
# Issue #3259585 by yaroslav-shandra: Private message text is rendered with html code

## Problem
When user join a Group (or a course or a challenge) you can setup a welcome message and add style for the message text.
![image](https://user-images.githubusercontent.com/80881183/150363585-eea5c5d3-b7be-4e97-a949-8fab6ea3a4ef.png)
The welcome email message is sent to users when they join/added to the group and don't login to the platform.
Users get an email with HTML code in the body message.
![image](https://user-images.githubusercontent.com/80881183/150363729-b77e8e52-f8ed-46aa-a32a-2a2fa355633e.png)

## Solution
Fix twig templates for a welcome message

## Issue tracker
https://www.drupal.org/project/social/issues/3259585
https://getopensocial.atlassian.net/browse/SUP-1997

## How to test
1. Check if enabled sub module Social group welcome message
2. Create a group with a welcome message
3. Add user to the group

## Screenshots
*If this Pull Request makes visual changes then please include some screenshots that show what has changed here. A before and after screenshot helps the reviewer determine what changes were made.*

## Release notes
Fixed  massage template for render private message text.
